test/scenarios/bookshop/funcs.test.js

```diff
@@ -288,9 +318,17 @@ describe('Bookshop - Functions', () => {
       expect(res.data.value.length).to.be.eq(1)
     })
     // REVISIT: does not seem database relevant
-    test.skip('time', () => { })
+    test.skip('time', async () => {
+      // Error on HANA with okra and new adapter: 328 - invalid name of function or procedure: TIME
+      // Works on SQLite
+      const res = await GET(`/browse/Books?$select=ID,createdAt&$filter=time(createdAt) gt 00:01:45&$top=1`)
+
+      expect(res.status).to.be.eq(200)
+      expect(res.data.value.length).to.be.eq(1)
+    })
     test.skip('totaloffsetminutes', async () => {
-      // REVISIT: ERROR: Feature is not supported: Method "totaloffsetminutes" in $filter or $orderby query options
+      // okra error: Feature is not supported: Method "totaloffsetminutes" in $filter or $orderby query options
+      // new adapter: no error, wrong result
```

Comment:
Should be double checked what the actual result is of the function for this value.

[totaloffsetminutes](https://github.com/cap-js/cds-dbs/blob/main/db-service/lib/cql-functions.js#L220)